rc/extra/editorconfig.kak

```diff
@@ -33,13 +45,19 @@ define-command editorconfig-load -params ..1 -docstring "editorconfig-load [file
                 if (charset == "utf-8-bom") {
                     print "set-option buffer BOM utf8"
                 }
+                if (insert_final_newline == "true") {
+                    print "set-option buffer editorconfig_insert_final_newline true"
+                }
                 if (trim_trailing_whitespace == "true") {
                     print "set-option buffer editorconfig_trim_trailing_whitespace true"
                 }
             }
         '
     }
     hook buffer BufWritePre %val{buffile} -group editorconfig-hooks %{ evaluate-commands %sh{
+        if [ ${kak_opt_editorconfig_insert_final_newline} = "true" ]; then
```

Comment:
I think you need to enclose this expansions in double quotes.

Reply:
not sure, line below is:
 `if [ ${kak_opt_editorconfig_trim_trailing_whitespace} = "true" ]; then`

Reply:
also shellcheck does not complain

Reply:
I guess it's fine because this option can only expand in a single alphabetical word, but usually arbitrary expansions have to be enclosed in double quotes.

Reply:
I know, I can change it for both lines to be consistent, wdyt?

Reply:
Leave it, it's not an issue for now.